error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/ExplicitArgumentEnumeration.java

```diff
@@ -213,9 +213,13 @@ private static boolean hasLikelySuitableVarargsOverload(
     // `X<T>`, where `T` is the type of the explicitly enumerated values passed to the expression to
     // be unwrapped. This should generally hold, given the types returned by the
     // `EXPLICIT_ITERABLE_CREATOR` expressions: `Iterable<T>`, `List<T>`, `Set<T>`, etc.
-    Type parameterType =
-        Iterables.getOnlyElement(
-            Iterables.getOnlyElement(method.getParameters()).type.getTypeArguments());
+    List<Type> typeArguments =
+        Iterables.getOnlyElement(method.getParameters()).type.getTypeArguments();
+    if (typeArguments.isEmpty()) {
+      // Can't determine the element type, so we can't suggest a fix.
+      return false;
+    }
```

Comment:
It seems to me that we can do better than this, namely by assuming a safe default: `java.lang.Object`.